lib/kubernetes-deploy/kubernetes_resource/custom_resource_definition.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class CustomResourceDefinition < KubernetesResource
+    TIMEOUT = 30.seconds
```

Comment:
I'd suggest being a little more generous here, since something actually does need to happen (vs. configmaps etc. where we're really just creating them and seeing that they showed up)